install.sh

```
@@ -81,107 +193,188 @@ if [ -d "$INSTALL_DIR" ]; then
+    fi
+  fi
+
+  echo
+  printf "  ${GREEN}${BOLD}Worker installed.${RESET}\n"
+  printf "  ${DIM}Paste these into Sirene → Settings → Inference servers → Add server:${RESET}\n"
+  echo
+  printf "    ${YELLOW}URL${RESET}        %s\n" "$SERVER_URL"
+  printf "    ${YELLOW}Auth token${RESET} %s\n" "$AUTH_TOKEN"
+  echo
+  printf "  ${DIM}Models:${RESET} %s\n" "$DATA_DIR_ABS"
+  printf "  ${DIM}Token:${RESET}  $INSTALL_DIR/auth_token\n"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)